django/db/backends/base/schema.py

```diff
@@ -302,7 +308,7 @@ def create_model(self, model):
         # Make the table
         sql = self.sql_create_table % {
             "table": self.quote_name(model._meta.db_table),
-            "definition": ", ".join(column_sqls)
+            "definition": ", ".join(chain(column_sqls, constraints))
```

Comment:
We probably can, but I don't think it's better.

Reply:
but it's faster because there's no function call overhead

```
In [1]: from itertools import chain

In [2]: a = ['1', '2']

In [3]: b = ['buckle', 'my', 'shoe']

In [4]: %timeit list(chain(a, b))
506 ns ± 2.19 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)

In [5]: %timeit list((*a, *b))
313 ns ± 4.46 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)
```

Reply:
Ok - I'll do it then.